app/views/appointments/index.csv.erb

```diff
@@ -0,0 +1,30 @@
+<%= [
+  "Patient name",
+  "Gender",
+  "Age",
+  "Days overdue",
+  "Last BP",
+  "Last BP taken at",
+  "Last BP date",
+  "Risk level",
+  "Patient address",
+  "Patient village or colony",
+  "Patient phone"
+].to_csv -%>
```

Comment:
Looks like `to_csv` adds a `\n` at the end of the line by default. This should be changed to `to_csv( row_sep: nil )`.